helpers/commonHelper.js

```diff
@@ -0,0 +1,10 @@
+import _ from 'lodash'
+
+export default {
+  isValidUuid (uuid = '') {
```

Comment:
I think that there should not be default value for uuid parameter because it would be false anyway